vidyut-prakriya/www/templates/index.html

```diff
@@ -255,6 +255,17 @@ <h1 class="text-4xl font-bold mb-2">{{ 'vidyullekhA'|d }}</h1>
     </p>
   </header>
 
+  <div class="max-w-3xl mx-auto text-right">
+    <select x-model="script" class="p-2 py-2 my-2">
```

Comment:
you can remove `py-2` since it is included in `p-2`.

Reply:
Aah! I understand now, my css-fu isn't great. Good to know! Will make the change and push.

Reply:
@akprasad Have made the change and refactored the `wasm-pack` check as suggested Gouenji in discord.